cpp/daal/src/externals/core_threading_win_dll.cpp

```diff
@@ -26,6 +26,7 @@
 #include "src/threading/threading.h"
 #include "src/threading/service_thread_pinner.h"
 #include "services/env_detect.h"
+#include "services/daal_defines.h"
```

Comment:
Not required here, `env_detect.h` already includes `daal_defines.h`

Reply:
Fixed,
thanks

Reply:
Why `mkl_daal.h` is deleted here? Is it needed or not? If it's not needed, why `service/daal_defines.h` is added?

My previous comment was about order of includes, it was
```c++
#include "services/env_detect.h"
#include "mkl_daal.h"
```

You add `service/daal_defines.h`, so it becomes
```
#include "services/env_detect.h"
#include "service/daal_defines.h"
#include "mkl_daal.h"
```

My comment was about `service/daal_defines.h`, it's not needed, because `services/env_detect.h` already includes it.